pomdp.py

```diff
@@ -163,19 +164,27 @@ def __get_value(self, i):
         return i + 1
 
     def __get_states(self, i):
-        # TODO support states as number
         line = self.contents[i]
         self.states = line.split()[1:]
+        if is_numeric(self.states):
+            no_states = self.states[0]
+            self.states = [str(x) for x in range(int(no_states))]
```

Comment:
For these three, cases, I'd either modify `no_` variables to be more clearly a number by changing the name and type, like:

```python
num_states = int(self.states[0])
```

or just remove it (since it's only used once) and instead have lines like:

```python
self.states = [str(x) for x in range(int(self.states[0]))]
```

your pick!